website/docs/topics/the-anatomy-of-unleash.mdx

```diff
@@ -0,0 +1,202 @@
+- [Strategy types](../user_guide/activation-strategies.md) (including [custom activation strategy types](../advanced/custom-activation-strategy.md))
+- [Tag types](../advanced/tags.md)
+- [Unleash context](../user_guide/unleash-context.md) fields (including [custom context fields](../user_guide/unleash-context.md#custom-context-fields))
+- Users, [user groups](../user_guide/rbac.md#user-groups) and [global roles](../user_guide/rbac.md)
+
+
+## Projects
+
+[**Projects**](../user_guide/projects.md) contain [feature toggles](../reference/feature-toggles.mdx) and their configurations, and a set of active [environments](../user_guide/environments.md).
+
+All Unleash instances must have at least one project at any given time. New instances get a project called “Default”.
+
+Pro and Enterprise customers can create, rename, and delete projects as they wish (as long as there is always **at least one project**). Open-source users, on the other hand, only get access to the Default project.
+
+<Figure caption="Unleash projects contain one or more environments." alt="A square labeled 'project' containing another square, labeled 'environment'." img="/img/anatomy-of-unleash-environment.png"/>
+
+## Environments and project environments
+
+<Figure img="/img/anatomy-of-unleash-customer-tiers.png" caption="Feature toggles can be activated or deactivated independently in different environments. For instance, a feature toggle can be active in the development environment, and deactivated in the production environment. Even if their configuration is otherwise identical, deactivated feature toggles will never be considered enabled."/>
+
+[**Environments**](../user_guide/environments.md) in Unleash let you change how a feature toggle works in your application’s different environments. For instance, while you are developing a feature, it’s likely that you’ll want it to be available in your development environment, but not in your production environment: environments let you do that. You might also want to enable a feature for only some users in your development environment, but no users in your production environment: environments let you do that.
+
+Environments exist on two different levels within Unleash. The set of **all available environments is defined on the global level**. Additionally, **each project** can choose which of these global environments should be **available on the project level**. The set of environments available to any given project is **always a subset** of the set of globally available environments.
+
+Each project must always have **at least one** active environment.
+
+Enterprise users can create and remove environments. Open-source and Pro customers get access to two environments: **development** and **production.**
+
+Environments are adjacent to [feature toggles](../reference/feature-toggles.mdx) in Unleash: neither one contains the other, but they come together to let you define activation strategies.
+
+:::tip Environments and API keys
```

Comment:
This "tip" section is a clarification you can put at the end of the Environments section. It's complementary info, that should not distract people in their cognitive effort to acquire the Environments concept. After they read 5 nice paragraphs about the topic (lines 38 to 45), they should immediately see the Project:Checkout illustration to fix those ideas ... the complementary info in the "tip" can come after the illustration, in my opinion.

Reply:
Yeah, I can agree with that. Would you keep the text within the admonition (the green background, the lightbulb, etc) or just make the text a regular paragraph?

Reply:
Thinking about it, I made it a blue info-admonition instead (different colors have different meanings). I think that makes more sense. Still happy to hear you out if you think it should just be a regular paragraph instead.